backend/engine/utils/plugin.py

```diff
@@ -460,7 +469,11 @@ def queue_event(repo, plugin_type, payload):
 
 
 def get_secret_raw_wl(scan):
```

Comment:
and here

Reply:
Same issue as above (and same implicit attribute which is the problem).